### PR TITLE
Frontend cleanup

### PR DESCRIPTION
**Pull request by FinalAngel**
- use correct spacing
- 80/120 line rule
- fix image issues

### CODE REVIEW

File: aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/plugins/list/standard/list.html [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
-</div>
+    <div class="aldryn aldryn-events aldryn-events-listplugin">
+        <ul class="list-unstyled">
+            {% for event in events %}
+                <li>
+                    <a href="{{ event.get_absolute_url }}">{{ event }}</a>
+                    <span class="date">{{ event.start|date }}</span>
+                </li>
+            {% empty %}
+                <li class="well">{% trans "No items available" %}</li>
```
Comment (vxsx) [inline aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/plugins/list/standard/list.html]: `No events available` maybe?

File: aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/plugins/upcoming/standard/upcoming.html [hunk 2]
```diff
@@ -2,19 +2,22 @@
...
+            {% endif %}
+        {% else %}
+            <ul class="list-unstyled">
+                {% for event in events %}
+                    <li>
+                        <a href="{{ event.get_absolute_url }}">{{ event }}</a>
+                        <span class="date">{{ event.start|date }}</span>
+                    </li>
+                {% empty %}
+                    <li class="well">{% trans "No items available" %}</li>
```
Comment (vxsx) [inline aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/plugins/upcoming/standard/upcoming.html]: same here

**Comment by vxsx** [comment]
codewise LGTM

File: aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/plugins/list/standard/list.html (hunk 1 above)
Comment (FinalAngel) [inline aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/plugins/list/standard/list.html]: It's named like this accross the addons

**Pull request merged**